tests/stepsDefinitions/HomeSteps.js

```diff
@@ -1,5 +1,5 @@
 import { Given, When, Then } from 'cucumber';
-import HomeScreen from '../screenObjects/HomeScreen.js';
+import HomeScreen from '../screenObjects/HomeScreen';
 
 Given(/^launch the app$/, () => {
   driver.launchApp();
```

Comment:
We should not use a step to launch the app. In most of scenarios (maybe all) we will need the app open. So probable open the app and set everything with hooks, and assume every tests starts with the app opened in the same screen.